ConnectorActionableMsgs/02 Cards and Actions/Demos/03-Expense-Approval/readme.md

```diff
@@ -0,0 +1,319 @@
+# Cards and Actions Using Outlook Actionable Messages – 300 Level
+----------------
+In this lab, you will walk through building an Actionable Message card and adding actions. 
+
+## Prerequisites
+
+This lab uses Visual Studio 2017. It also requires an Office 365 subscription with an active mailbox and a **Microsoft Azure** subscription.
+
+## Setup
+
+This lab will use an Azure Web App to deploy an application. The URL of the web app is required. Visit the [Microsoft Azure Portal](https://portal.azure.com) and create a new Web App. Copy the URL (for example, https://mywebapp.azurewebsites.net) for later use.
+
+
+## Adding actions to cards
+
+The first section of this lab demonstrated how to design a card, the second section demonstrated how to send Actionable Messages. This section will now pull it all together by implementing a Web API that responds to card actions.
+
+## Register a new provider
+As you saw previously in this lab, the action URL did not yet work because it had not been whitelisted. Registration of a provider is required to whitelist your action URL to use with cards. 
```

Comment:
Actually this isn't true. You don't need to register to send actionable email to yourself. As long as you have the backend service running, you can invoke actions fine. I'd cut this.